assets/js/components/Sectors.vue

```diff
@@ -408,13 +441,13 @@ export default Vue.extend({
     },
 
     // WIP - reset the chart
-    // reset(){
-
-    // const $this = this;
-    // const arc = $this.svg.select("g.chart").selectAll(".arc").remove();
-    // const label = $this.svg.select("g.legend").selectAll(".label").remove();
-    //  $this.doStuff();
-    // },
+    // TODO - make this smoother
+    reset(){
```

Comment:
this is mean.
you just have to do all that same stuff, but in reverse. it should be simple, right? :)
especially if the code is clean and readable. ;)